doc/source/reference/c-api/array.rst

```diff
@@ -3123,15 +3125,15 @@ the C-API is needed then some additional steps must be taken.
     Internally, these #defines work as follows:
 
         * If neither is defined, the C-API is declared to be
-          :c:type:`static void**`, so it is only visible within the
+          ``static void**``, so it is only visible within the
           compilation unit that #includes numpy/arrayobject.h.
         * If :c:macro:`PY_ARRAY_UNIQUE_SYMBOL` is #defined, but
           :c:macro:`NO_IMPORT_ARRAY` is not, the C-API is declared to
-          be :c:type:`void**`, so that it will also be visible to other
+          be ``void**``, so that it will also be visible to other
```

Comment:
I wonder if ``` :c:expr:`void **` ``` does something sensible here

Reply:
There're ``static void**`` and ``extern void**`` around here, and when I rewrote using ``:c:expr:`` the below is raised:
```
WARNING: Unparseable C expression: 'static void**'
Error when parsing (type) expression.
If expression:
  Invalid C declaration: Expected identifier in nested name, got keyword: static [error at 6]
    static void**
    ------^
If type:
  Invalid C declaration: Expected identifier in nested name, got keyword: static [error at 6]
    static void**
    ------^
```
and same for ``extern``. 

It seems strange to apply ``:c:expr:`` only for ``void **``.